### DISCUSSION TITLE
IC-40 Super Chlorinate

### DISCUSSION THREAD

**Discussion by dcorrea78** [tagyoureit/nodejs-poolController#789] [App and O/S Config Questions]
Is there a way to start the super chlorinate feature from the dashpanel? If I start super chlorinate from the Pentair app, the dash panel displays super chlorinate and the time remaining, but I cannot figure out any other way to start super chlorinate without going into the Pentair app first.

**Comment by dcorrea78** [comment]
Disregard. Turns out the pop-up to enable super chlorinate doesn't work on my Android browser. On my laptop it works fine.

**Reply by rstrouse** [reply]
Update dashPanel as this was likely fixed last week.
